src/Raven.Client/Documents/Queries/RavenQuery.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+
+namespace Raven.Client.Documents.Queries
+{
+    public class RavenQuery
+    {
+        public static T Load<T>(string id)
+        {
+            throw new NotSupportedException("Never called");
```

Comment:
Need to say something like: "This method is here for strongly type support of server side call during Linq queries and should never be directly called"